src/redux/wallets.js

```diff
@@ -212,6 +214,7 @@ export const createAccountForWallet = (id, color, name) => async (
   await saveAddress(account.address);
   // Set the wallet selected (KEYCHAIN)
   await setSelectedWallet(newWallets[id]);
+  await saveAccountEmptyState(true, account.address, network);
```

Comment:
We forgot to flag the account as empty on creation of a new wallet

Reply:
this looks like its for loading existing wallets, so we want it to be not be flagged. new wallets are created using `createWallet` 

@jinchung for xtra eyes

Reply:
Actually, disregard, we don't need this anymore